docs/src/content/faq.mdx

```diff
@@ -48,27 +81,275 @@ After receiving the NFT, follow these steps to set up the Icebreaker platform:
+
+A future update will allow dynamic IPs by having the node proactively report its address. See [this GitHub issue](https://github.com/blockfrost/blockfrost-platform/issues/178) for details on that roadmap.
+
+In the interim, using a static IP is best. If not possible, a dynamic DNS service or VPN may help provide a stable hostname to IP mapping.
+
+### What should I do if my public IP address changes frequently?
+
+Frequent public IP address changes can disrupt the connection between your Icebreaker and the Blockfrost servers.
+A future update will address this by allowing connections from behind NATs (see [this GitHub Issue](https://github.com/blockfrost/blockfrost-platform/issues/178)).
+In the meantime, consider using a static IP address or a dynamic DNS service.
+
+### How do I run blockfrost-platform with only IPv4?
+
+While a dedicated `-4` CLI switch isn't currently available, you can work around IPv6 issues by adding an entry to your `/etc/hosts` file that maps the Blockfrost server address to its IPv4 address.
+
+### Should I run my Icebreaker node on bare metal or in a VM?
+
+Either bare metal or VM-based Icebreaker nodes will work fine.
+The choice mainly depends on your available infrastructure and operational preferences.
+
+Benefits of bare metal:
+- Avoids virtualization overhead for maximum performance
+- Provides direct access to hardware for optimization
+- Keeps node infrastructure separate from other workloads
+
+Benefits of VMs:
+- Easier to deploy and manage using common VM platforms
+- Allows sharing server resources with other workloads
+- Provides snapshot and migration capabilities for easier updates
+
+Regardless of your approach, refer to the [Getting started](get-started) documentation to check the prerequisites.
+Over-provisioning CPU, RAM and disk will give you more headroom.
+
+### What is the Blockfrost Platform `server-address` configuration parameter used for?
+
+The `server-address` configuration parameter defines the IP address and port that your Icebreaker node's API server listens on.
+
+By default this uses `0.0.0.0:3000` which listens on all IPv4 interfaces on port 3000.
+
+Common customizations:
+
+- To use IPv6, specify `::` as the IP
+- To limit to localhost only, use `127.0.0.1`
+- To use a different port, change the `:3000` portion
+
+Be cautious when changing this value as it can impact your node's public reachability for API requests.
+Always ensure the specified interface and port are publicly accessible.
+
+## Operation and monitoring
+
+### How can I monitor and verify that my Icebreaker node is working correctly?
+
+Please see the [Verifying that you are up and running](/installation/verification) guide.
+
+### How can I check if my Icebreaker node is active?
+
+To verify if your Icebreaker node is active, you can:
+
+1. Access the [Icebreaker dashboard](https://blockfrost.grafana.net/public-dashboards/8d618eda298d472a996ca3473ab36177) to check the status of your node.
```

Comment:
1 and 2 sound like sequential checks, but they are independent of each other, right? It would be nice to make it clearer.

Also, how do I know which node is mine in the grafana dashboard?

Last thing: Just 1 minute ago, grafana dashboard can now be accessed via  http://icebreakers.blockfrost.io